### DISCUSSION TITLE
我是否可以不通过命令安装，直接复制到theme文件夹

### DISCUSSION THREAD

**Discussion by lem0nb** [ppoffice/hexo-theme-icarus#878] [Q&A] [answered]
> 确保你在提交功能建议之前仔细阅读了[Hexo文档](https://hexo.io/zh-cn/)，[Icarus用户指南](https://ppoffice.github.io/hexo-theme-icarus/tags/Icarus%E7%94%A8%E6%88%B7%E6%8C%87%E5%8D%97/)，和[GitHub issues](https://github.com/ppoffice/hexo-theme-icarus/issues)来了解你的建议是否已经被他人提出过。

**你的功能建议与某个使用问题相关么？请详述。**

简洁清晰地描述你遇到的问题是什么（如：我在使用...的时候遇到了...）。

**描述你想要的解决方案**
直接复制到theme文件夹
简洁清晰地描述你想要的解决方案可以达到的效果。

**描述你考虑过的替代办法**

简洁清晰地描述你考虑过的替代解决方案或是新功能。

**额外上下文**

附上与功能请求有关的其他上下文信息或者截图。

**Comment by ppoffice** [comment, marked as answer]
@lem0nb 可以，但是不推荐这么做。方法与直接从源码安装相似。

https://ppoffice.github.io/hexo-theme-icarus/uncategorized/icarus%E5%BF%AB%E9%80%9F%E4%B8%8A%E6%89%8B/
